blocks/editable/format-toolbar/index.js

```diff
@@ -40,22 +38,9 @@ const DEFAULT_CONTROLS = [ 'bold', 'italic', 'strikethrough', 'link' ];
 class FormatToolbar extends Component {
```

Comment:
I guess we could refactor this component to be just a functional component?

Reply:
I don't think we can as it needs the lifecycle methods?